code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -120,10 +122,12 @@
 			cached_gases[/datum/gas/oxygen][MOLES] -= cached_gases[/datum/gas/tritium][MOLES]
 
 		if(burned_fuel)
-			energy_released += FIRE_CARBON_ENERGY_RELEASED * burned_fuel
+			energy_released += FIRE_HYDROGEN_ENERGY_RELEASED * burned_fuel
+			if(location)
+				radiation_pulse(location, energy_released/TRITIUM_BURN_RADIOACTIVITY_FACTOR)
```

Comment:
ok, let's assume we burn 1 mole of tritium in very oxygen rich enviroment.
tritium burns at 10x its molar mass in oxygen, so that's 10 moles worth of burn
burning one mole produces FIRE_HYDROGEN_ENERGY_RELEASED energy (280000)
burning 10 moles thus produces 2.8 million whatevers
the energy released divided by 1000 returns the strength of the rad pulse

tl;dr one mole of tritium produces 2800 rads

Reply:
Okay, I sorta eyeballed it but I might've eyeballed it a bit too high. Let me test ingame and see how it goes.
But I asked coderbus, and they said it should have a lot of rads compared to the SM so...